app/mint/list/page.tsx

```diff
@@ -1,12 +1,36 @@
-import { Header } from "@/components/header";
-import { Separator } from "@/components/separator";
+"use client";
+
+import useSWR from "swr";
+import type { ApiMint, Mint } from "@/app/api/mints/route";
+import { MintCard } from "@/app/mint/list/mint-card";
+import { PageContainer } from "@/components/page-container";
+import { Skeleton } from "@/components/ui/skeleton";
+
+const mapApiMintsToMints = (data: ApiMint[]): Mint[] => {
+  return data.map((mint) => ({
+    ...mint,
+    tags: JSON.parse(mint.tags),
+    metadata: JSON.parse(mint.metadata),
+  }));
+};
+
+const fetcher = (url: string) =>
+  fetch(url)
+    .then((res) => res.json())
+    .then((json) => mapApiMintsToMints(json));
 
 export default function ListMints() {
+  const { data, isLoading, error } = useSWR("/api/mints", fetcher);
+
   return (
-    <>
-      <Header label="Minted Assets" />
-      <Separator />
-      <section>Placeholder</section>
-    </>
+    <PageContainer label="Minted Assets">
```

Comment:
We're wrapping most pages in this `PageContainer` - it might be worth looking to see if you can move that into a higher level root `layout.tsx` file?